circle.yml

```diff
@@ -35,3 +36,45 @@ jobs:
             chisel run
             ls -l target/runevm.wasm
             test -f target/runevm.wasm
+  test:
+    docker:
+      - image: ethereum/cpp-build-env:9
```

Comment:
Is this the latest?

Reply:
I had just copied from hera and hadn't looked :sweat_smile: updated to latest